ajo/personalization-client-side/README.md

```diff
@@ -31,7 +31,20 @@ To run this sample:
 
 ## How it works
 
-1. [Web SDK](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) is included on the page.
+1. [Web SDK](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) is included and configured on the page. The configuration is based on the `.env` file under ajo.
```

Comment:
I suggest rewriting this sentence to be more specific so the reader knows what "under ajo" means.  For example:

```
The configuration is based on the `.env` file within the `ajo` folder.
```